src/wayscript/integrations/sql/postgres.py

```diff
@@ -0,0 +1,44 @@
+import json
```

Comment:
How do we want to handle/think about different types of databases? Should I implement the equivalent of a `postgres` module for every database type? Which do we require for launch? I can ask Nihar about this.

Reply:
I think asking Nihar is a good idea, lets keep the scope of this or to postgres and create followups with any other types if they want them.